frontend/admin/src/js/components/user/UserTable.tsx

```diff
@@ -129,41 +201,7 @@ export const UserTable: React.FC = () => {
 
   const handleFilterSubmit: SubmitHandler<FormValues> = (data) => {
     // this state lives in the UserTable component, this step also acts like a formValuesToSubmitData function
-    setUserFilterInput({
-      applicantFilter: {
-        expectedClassifications: data.classifications.map((classification) => {
-          const splitString = classification.split("-");
-          return { group: splitString[0], level: Number(splitString[1]) };
-        }),
-        languageAbility: data.languageAbility[0]
-          ? stringToEnumLanguage(data.languageAbility[0])
-          : undefined,
-        locationPreferences: data.workRegion.map((region) => {
-          return stringToEnumLocation(region);
-        }),
-        operationalRequirements: data.operationalRequirement.map(
-          (requirement) => {
-            return stringToEnumOperational(requirement);
-          },
-        ),
-        skills: data.skills.map((skill) => {
-          const skillString = skill;
-          return { id: skillString };
-        }),
-        wouldAcceptTemporary: data.employmentDuration[0]
-          ? data.employmentDuration[0] === "TERM"
-          : undefined,
-      },
-      isGovEmployee: data.govEmployee[0] ? true : undefined,
-      isProfileComplete: data.profileComplete[0] ? true : undefined,
-      jobLookingStatus: data.jobLookingStatus.map((status) => {
-        return stringToEnumJobLooking(status);
-      }),
-      poolFilters: data.pools.map((pool) => {
-        const poolString = pool;
-        return { poolId: poolString };
-      }),
-    });
+    setUserFilterInput(formValuesToUserFilterInput(data));
```

Comment:
Starting functions with verbs?

```suggestion
    setUserFilterInput(transformUserFormValuesToFilterInput(data));
```